build-heads.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+build_only="${1}"
```

Comment:
With `set -u` this probably needs to be `${1:-}` since `$1` is undefined unless the user gave the command line argument.